immich: 1.131.3 -> 1.132.1

dotlambda opened the pull request:
Diff: https://github.com/immich-app/immich/compare/refs/tags/v1.131.3...refs/tags/v1.132.1

Changelog:
https://github.com/immich-app/immich/releases/tag/v1.132.0
https://github.com/immich-app/immich/releases/tag/v1.132.1

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda committed:
immich: 1.131.3 -> 1.132.1

Diff: https://github.com/immich-app/immich/compare/refs/tags/v1.131.3...refs/tags/v1.132.1

Changelog: https://github.com/immich-app/immich/releases/tag/v1.132.0
           https://github.com/immich-app/immich/releases/tag/v1.132.1

Scrumplex commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 401191`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>immich</li>
    <li>immich-cli</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
  </ul>
</details>

Scrumplex reviewed (approved):
NixOS test passes on x86_64-linux

Scrumplex commented:
https://github.com/immich-app/immich/releases/tag/v1.132.1

dotlambda committed:
vips: 8.16.0 -> 8.16.1

Diff: https://github.com/libvips/libvips/compare/refs/tags/v8.16.0...refs/tags/v8.16.1

Changelog: https://github.com/libvips/libvips/blob/refs/tags/v8.16.1/ChangeLog

dotlambda commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 401191`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>btc-rpc-explorer</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>python313Packages.pyvista</li>
    <li>python313Packages.pyvista.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 55 packages built:</summary>
  <ul>
    <li>chatd</li>
    <li>ente-desktop</li>
    <li>factoriolab</li>
    <li>faircamp</li>
    <li>gancio</li>
    <li>haskellPackages.gi-vips</li>
    <li>haskellPackages.gi-vips.doc</li>
    <li>imaginary</li>
    <li>imgproxy</li>
    <li>immich</li>
    <li>immich-cli</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>joplin</li>
    <li>lemmy-ui</li>
    <li>librechat</li>
    <li>mastodon</li>
    <li>netlify-cli</li>
    <li>nip2</li>
    <li>pds</li>
    <li>photoprism</li>
    <li>pingvin-share.backend</li>
    <li>pingvin-share.frontend</li>
    <li>pleroma</li>
    <li>previewqt</li>
    <li>psitransfer</li>
    <li>python312Packages.pyvips</li>
    <li>python312Packages.pyvips.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.scooby</li>
    <li>python312Packages.scooby.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python313Packages.pyvips</li>
    <li>python313Packages.pyvips.dist</li>
    <li>python313Packages.scooby</li>
    <li>python313Packages.scooby.dist</li>
    <li>qq</li>
    <li>rubyPackages.ruby-vips (rubyPackages_3_3.ruby-vips)</li>
    <li>rubyPackages_3_1.ruby-vips</li>
    <li>rubyPackages_3_2.ruby-vips</li>
    <li>rubyPackages_3_4.ruby-vips</li>
    <li>snapweb</li>
    <li>ueberzugpp</li>
    <li>vips</li>
    <li>vips.bin</li>
    <li>vips.dev</li>
    <li>vips.devdoc</li>
    <li>vips.man</li>
    <li>vipsdisp</li>
    <li>walker</li>
    <li>waveterm</li>
    <li>ytfzf</li>
    <li>zipline</li>
  </ul>
</details>

Scrumplex commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 401191`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>btc-rpc-explorer</li>
    <li>python313Packages.pyvista</li>
    <li>python313Packages.pyvista.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 55 packages built:</summary>
  <ul>
    <li>chatd</li>
    <li>ente-desktop</li>
    <li>factoriolab</li>
    <li>faircamp</li>
    <li>gancio</li>
    <li>haskellPackages.gi-vips</li>
    <li>haskellPackages.gi-vips.doc</li>
    <li>imaginary</li>
    <li>imgproxy</li>
    <li>immich</li>
    <li>immich-cli</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>joplin</li>
    <li>lemmy-ui</li>
    <li>librechat</li>
    <li>mastodon</li>
    <li>netlify-cli</li>
    <li>nip2</li>
    <li>pds</li>
    <li>photoprism</li>
    <li>pingvin-share.backend</li>
    <li>pingvin-share.frontend</li>
    <li>pleroma</li>
    <li>previewqt</li>
    <li>psitransfer</li>
    <li>python312Packages.pyvips</li>
    <li>python312Packages.pyvips.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.scooby</li>
    <li>python312Packages.scooby.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python313Packages.pyvips</li>
    <li>python313Packages.pyvips.dist</li>
    <li>python313Packages.scooby</li>
    <li>python313Packages.scooby.dist</li>
    <li>qq</li>
    <li>rubyPackages.ruby-vips (rubyPackages_3_3.ruby-vips)</li>
    <li>rubyPackages_3_1.ruby-vips</li>
    <li>rubyPackages_3_2.ruby-vips</li>
    <li>rubyPackages_3_4.ruby-vips</li>
    <li>snapweb</li>
    <li>ueberzugpp</li>
    <li>vips</li>
    <li>vips.bin</li>
    <li>vips.dev</li>
    <li>vips.devdoc</li>
    <li>vips.man</li>
    <li>vipsdisp</li>
    <li>walker</li>
    <li>waveterm</li>
    <li>ytfzf</li>
    <li>zipline</li>
  </ul>
</details>
